### ISSUE TITLE
Implement useful data into Resource Search and Details

### ISSUE THREAD

**Issue by mowmowkittycat** [SpiGetOrg/Spiget#54]
Resource Search and just getting Resource Details including Resource Author Id and Resource Author Version Id but dont actually include the Resource Author and the Resource Version. You have to make 2 seperate get requests if you want both of these, which is absolutely ridiculous. I am trying to make a package manager and I dont want to make 2 get requests per Resource for searching through resources and displaying version and author. The Official Spigot API gives you the version and author when listing resources why cant this do that.

Basically there shouldnt be sections of data that have just their id, you should provide the id and the data of that object or section.

**Comment by InventivetalentDev** [comment]
resources, authors, versions, etc. are all stored in separate collections internally, which is why requests for resources only return the reference to those other objects.  
If you do need the details for them, you just make a separate request - I don't really see why that's a huge issue.
